src/app/core/library-module/library.service.ts

```diff
@@ -86,14 +89,14 @@ export class LibraryService {
     }
     this.http
       .delete(
-        USER_ROUTES.CLEAR_LEARNING_OBJECT_FROM_CART(
+        LIBRARY_ROUTES.REMOVE_LEARNING_OBJECT_FROM_LIBRARY(
           this.user.username,
           cuid
```

Comment:
The backend is expecting the libraryId not cuid. Also in the routes file it says learningObjectId